oneBarangay/static/assets/js/volt.js

```diff
@@ -260,96 +260,96 @@ d.addEventListener("DOMContentLoaded", function (event) {
-        button.style.display = "none";
-        allLoaded.style.display = "block";
-      }, 1500);
-    });
+        loadContent.style.display = 'block'
+        button.style.display = 'none'
+        allLoaded.style.display = 'block'
+      }, 1500)
+    })
   }
 
   const scroll = new SmoothScroll('a[href*="#"]', {
     speed: 500,
-    speedAsDuration: true,
-  });
+    speedAsDuration: true
+  })
 
-  if (d.querySelector(".current-year")) {
-    d.querySelector(".current-year").textContent = new Date().getFullYear();
+  if (d.querySelector('.current-year')) {
+    d.querySelector('.current-year').textContent = new Date().getFullYear()
   }
 
   // Glide JS
 
-  if (d.querySelector(".glide")) {
-    new Glide(".glide", {
-      type: "carousel",
+  if (d.querySelector('.glide')) {
+    new Glide('.glide', {
+      type: 'carousel',
       startAt: 0,
-      perView: 3,
-    }).mount();
+      perView: 3
+    }).mount()
   }
 
-  if (d.querySelector(".glide-testimonials")) {
-    new Glide(".glide-testimonials", {
-      type: "carousel",
+  if (d.querySelector('.glide-testimonials')) {
+    new Glide('.glide-testimonials', {
+      type: 'carousel',
       startAt: 0,
       perView: 1,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-clients")) {
-    new Glide(".glide-clients", {
-      type: "carousel",
+  if (d.querySelector('.glide-clients')) {
+    new Glide('.glide-clients', {
+      type: 'carousel',
       startAt: 0,
       perView: 5,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-news-widget")) {
-    new Glide(".glide-news-widget", {
-      type: "carousel",
+  if (d.querySelector('.glide-news-widget')) {
+    new Glide('.glide-news-widget', {
+      type: 'carousel',
       startAt: 0,
       perView: 1,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-autoplay")) {
-    new Glide(".glide-autoplay", {
-      type: "carousel",
+  if (d.querySelector('.glide-autoplay')) {
+    new Glide('.glide-autoplay', {
+      type: 'carousel',
       startAt: 0,
       perView: 3,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
   // Pricing countup
-  const billingSwitchEl = d.getElementById("billingSwitch");
+  const billingSwitchEl = d.getElementById('billingSwitch')
   if (billingSwitchEl) {
-    const countUpStandard = new countUp.CountUp("priceStandard", 99, { startVal: 199 });
-    const countUpPremium = new countUp.CountUp("pricePremium", 199, { startVal: 299 });
+    const countUpStandard = new countUp.CountUp('priceStandard', 99, { startVal: 199 })
+    const countUpPremium = new countUp.CountUp('pricePremium', 199, { startVal: 299 })
 
-    billingSwitchEl.addEventListener("change", function () {
+    billingSwitchEl.addEventListener('change', function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
        billingSwitchEl.addEventListener("change", function() {
```